src/strings/strings-en.ts

```diff
@@ -735,6 +737,10 @@ export const strings = {
   REPORT_COULD_NOT_UNLOCK: 'Could not unlock report',
   REPORT_EDIT: 'Edit Report',
   REPORT_NAME: 'Report Name',
+  REPORT_NURSERY_BUILD_COMPLETION_DATE: 'Nursery Build Completion Date *',
+  REPORT_NURSERY_BUILD_START_DATE: 'Nusery Build Start Date *',
```

Comment:
typo `Nusery`.. strangely I often end up with this typo..

Reply:
Good catch! Thanks!